### PR TITLE
maigret url; pkgver/pkgrel update

### PR DESCRIPTION
**Pull request by dualfade**
updates as requested.

### CODE REVIEW

File: packages/maigret/PKGBUILD [hunk 1]
```diff
@@ -2,16 +2,16 @@
 # See COPYING for license details.
 
 pkgname=maigret
-pkgver=1278.7ffe4be
-pkgrel=1
+pkgver=1279.b577477
+pkgrel=2
```
Comment (Edu4rdSHL) [inline packages/maigret/PKGBUILD:6]: Don't bump pkgrel if pkgver was bumped.

File: packages/maigret/PKGBUILD [hunk 2]
```diff
@@ -36,9 +36,9 @@ package() {
 #!/bin/sh
 cd /usr/share/$pkgname
 source venv/bin/activate
-exec python $pkgname/$pkgname.py "\$@"
+exec python maigret/maigret.py "\$@"
 EOF
 
-  chmod a+x "$pkgdir/usr/bin/$pkgname"
+chmod a+x "$pkgdir/usr/bin/$pkgname"
 }
```
Comment (Edu4rdSHL) [inline packages/maigret/PKGBUILD:43]: Remove spaces in this closing bracket.

**Review by Edu4rdSHL — CHANGES_REQUESTED** [review]
Hey @dualfade thanks!

File: packages/maigret/PKGBUILD (hunk 1 above)
Comment (dualfade) [reply packages/maigret/PKGBUILD:6]: Adjusting.

File: packages/maigret/PKGBUILD (hunk 2 above)
Comment (dualfade) [reply packages/maigret/PKGBUILD]: Not sure. Here. My editor may have done this ? Lmk if it's ok now. Odd.

**Pull request closed without merging**